README.md

```diff
@@ -30,29 +30,22 @@
 - [🙏 Acknowledgements](#acknowledgements)
 - [📝 License](#license)
 
-# 📖 Math magicians <a name="about-project"></a>
+# 📖 Book Store <a name="about-project"></a>
 
-**Math magicians** is a single page app that uses api for a basic calculator.
+**Book Store** is a single page app that lets user add and remove books.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-- **HTML, CSS, React, and JavaScript**
+- **HTML, React, and JavaScript**
 
 ### Key Features <a name="key-features"></a>
 
-- **html styled with css and javascript**
 - **uses react**
```

Comment:
- [ ] [optional] It would be nice if you were more specific about the features you added in this milestone.